pnpm-workspace.yaml

```diff
@@ -1,5 +1,3 @@
 packages:
   - 'docs'
   - 'packages/*'
-  - 'ts-tests'
-  - 'meta'
```

Comment:
Why is 'meta' removed? It shouldn't be removed.

Reply:
added back in.